tests/fixtures/saas_fixtures.py

```diff
@@ -99,6 +111,112 @@ def connection_config_saas(
     connection_config.delete(db)
 
 
+@pytest.fixture(scope="function")
+def erasure_policy_string_rewrite(
```

Comment:
It would be useful for these policies to live alongside the others in `application_fixtures.py`, since they could be used in any test in the application and not only in the SaaS integration tests. That way anyone looking at policies will see every policy we have, and be less likely to add a new one if it's not needed.

Reply:
Good idea, moved these to `application_fixtures.py`